adoc/admin-security-ldap.adoc

```diff
@@ -47,72 +45,43 @@ To replace the automatically generated certificate with your own, follow these s
 docker stop 389-ds
 ----
 
-. Copy the external certificate `<rootCA.pem>` to a mounted data directory `<DATA_DIR>/config/`.
+. Copy the external private key `Server-Cert.crt`, certificate `Server-Cert.csr`, and password `pwdfile-import.txt` to a mounted data directory `<DATA_DIR>/config/`.
 
 . Run bash in entrypoint to access the container with the mounted data directory from the previous step:
 +
 ----
 docker run --rm -it \
-  -p 389:389 \
-  -p 636:636 \
   -v <DATA_DIR>:/data \
   --entrypoint /bin/bash \
-  --name 389-ds registry.suse.com/caasp/v4/389-ds:1.4.0
+  --name 389-ds-cert registry.suse.com/caasp/v4/389-ds:1.4.0
```

Comment:
I thought we get a new image which includes the fixed entry point? In this case the image version should not be 1.4.0 anymore as this is the broken one.

Reply:
This PR is mean to update the document for using old 389ds image.

Reply:
Ok - could you help me pointing me to the PR for using the new image?